### PR TITLE
Added hashed support ,Updated Readme

### LINKED ISSUE #304: Update usage of hash library in documentation
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file README.md in the `leaflet-environmental-layers` repository and edit with the information given below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

We are now using [leaflet-fullUrlHash.js](https://github.com/publiclab/leaflet-environmental-layers/blob/master/lib/leaflet-fullUrlHash.js) to generate URL hash. We now add the following after initializing the map: `var hash = new L.FullHash(map, allMapLayers);`

The documentation needs to be updated here:
https://github.com/publiclab/leaflet-environmental-layers/blob/master/README.md#add-leaflet-fullhash-for-easy-sharing-of-map- 

with the following:
```
## Add hash support for easy sharing of map :    

### Steps To add :

1.) Add link :

   <script src="../lib/leaflet-fullUrlHash.js"></script>

2.) Once you have initialized the map (an instance of L.Map), add the following code :

    // Assuming your map instance is in a variable called map and an object with all the map layers is 
    in a variable called allMapLayers
    var hash = new L.FullHash(map, allMapLayers);    
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by Marcus-auz**
Fixes #304  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by cesswairimu** [comment]
Hi @Marcus-auz , thank for opening this..kindly add the issue number on the pull request where you have `Fixes #issue_number`

**Comment by Marcus-auz** [comment]
Added

File: README.md [hunk 1]
```diff
@@ -214,18 +214,19 @@ We're going to try spinning this out into its own library; see: https://github.c
...
 
-## Add Leaflet-FullHash for easy sharing of map :    
+## Add hash support for easy sharing of map :    
 
 ### Steps To add :
 
 1.) Add link :
 
-    <script src="../node_modules/leaflet-fullhash/leaflet-fullHash.js"></script>
+       <script src="../lib/leaflet-fullUrlHash.js"></script>
```
Comment (crisner) [inline README.md]: I am not sure if I am being picky about the extra space added before the script tag. Other than that the changes look good to me. What do you think @sagarpreet-chadha?

**Comment by crisner** [comment]
Thanks for working on this @Marcus-auz! Let's wait for @sagarpreet-chadha's feedback. :)

**Comment by Marcus-auz** [comment]
sure sir

**Pull request merged**